Packs/Trello/IncidentTypes/Trello.json

```diff
@@ -0,0 +1,168 @@
+		"version": 3,
+		"modified": "2021-02-15T23:00:50.173812343Z",
+		"sortValues": null,
+		"packID": "",
+		"itemVersion": "",
+		"fromServerVersion": "",
+		"toServerVersion": "",
+		"vcShouldIgnore": false,
+		"vcShouldKeepItemLegacyProdMachine": false,
+		"commitMessage": "",
+		"shouldCommit": false,
+		"locked": false,
+		"name": "Engineering Review",
+		"prevName": "Engineering Review",
+		"color": "#80DEEA",
+		"hours": 0,
+		"days": 0,
+		"weeks": 0,
+		"hoursR": 0,
+		"daysR": 0,
+		"weeksR": 0,
+		"system": false,
+		"readonly": false,
+		"default": false,
+		"autorun": false,
+		"preProcessingScript": "",
+		"closureScript": "",
+		"disabled": false,
+		"reputationCalc": 0,
+		"layout": "a81e5644-f60a-477c-8fed-4e7e08532565"
+	},
+	{
+		"id": "LOLBIN",
+		"version": 1,
+		"modified": "2020-12-22T23:30:06.135462787Z",
+		"sortValues": null,
+		"packID": "",
+		"itemVersion": "",
+		"fromServerVersion": "",
+		"toServerVersion": "",
+		"vcShouldIgnore": false,
+		"vcShouldKeepItemLegacyProdMachine": false,
+		"commitMessage": "",
+		"shouldCommit": false,
+		"locked": false,
+		"name": "LOLBIN",
+		"prevName": "LOLBIN",
+		"color": "#7C71F5",
+		"hours": 0,
+		"days": 0,
+		"weeks": 0,
+		"hoursR": 0,
+		"daysR": 0,
+		"weeksR": 0,
+		"system": false,
+		"readonly": false,
+		"default": false,
+		"autorun": false,
+		"preProcessingScript": "",
+		"closureScript": "",
+		"disabled": false,
+		"reputationCalc": 0,
+		"layout": ""
+	},
+	{
+		"id": "Logrythm Generic",
+		"version": 1,
+		"modified": "2020-12-22T23:31:44.659032409Z",
+		"sortValues": null,
+		"packID": "",
+		"itemVersion": "",
+		"fromServerVersion": "",
+		"toServerVersion": "",
+		"vcShouldIgnore": false,
+		"vcShouldKeepItemLegacyProdMachine": false,
+		"commitMessage": "",
+		"shouldCommit": false,
+		"locked": false,
+		"name": "Logrythm Generic",
+		"prevName": "Logrythm Generic",
+		"color": "#5C6BC0",
+		"hours": 0,
+		"days": 0,
+		"weeks": 0,
+		"hoursR": 0,
+		"daysR": 0,
+		"weeksR": 0,
+		"system": false,
+		"readonly": false,
+		"default": false,
+		"autorun": false,
+		"preProcessingScript": "",
+		"closureScript": "",
+		"disabled": false,
+		"reputationCalc": 0,
+		"layout": ""
+	},
+	{
+		"id": "Privilege Escalation",
+		"version": 1,
+		"modified": "2020-12-22T23:30:20.323666833Z",
+		"sortValues": null,
+		"packID": "",
+		"itemVersion": "",
+		"fromServerVersion": "",
+		"toServerVersion": "",
+		"vcShouldIgnore": false,
+		"vcShouldKeepItemLegacyProdMachine": false,
+		"commitMessage": "",
+		"shouldCommit": false,
+		"locked": false,
+		"name": "Privilege Escalation",
+		"prevName": "Privilege Escalation",
+		"color": "#536DFE",
+		"hours": 0,
+		"days": 0,
+		"weeks": 0,
+		"hoursR": 0,
+		"daysR": 0,
+		"weeksR": 0,
+		"system": false,
+		"readonly": false,
+		"default": false,
+		"autorun": false,
+		"preProcessingScript": "",
+		"closureScript": "",
+		"disabled": false,
+		"reputationCalc": 0,
+		"layout": ""
+	},
+	{
+		"id": "Trello",
```

Comment:
@adambaumeister There are a few types here, I suppose you want to keep only the last one right?